core/src/query.rs

```diff
@@ -234,14 +235,14 @@ pub fn read(name: &str, vt: i32, limit: i32) -> Result<String, PgmqError> {
         (
             SELECT msg_id
             FROM {PGMQ_SCHEMA}.{TABLE_PREFIX}_{name}
-            WHERE vt <= now()
+            WHERE vt <= clock_timestamp()
```

Comment:
`now()` uses transaction start time. It's pretty useful, but in this case, if we repeatedly poll it will always query with the same timestamp. I also tried `statement_timestamp()`, but for some reason while using the SPI I got the same timestamp for different runs 🤔, so I resorted to `clock_timestamp()`